packages/itwin/breakdown-trees/src/Views/TreeNodeFunctionsToolbar.tsx

```diff
@@ -25,23 +25,22 @@ export const TreeNodeFunctionsToolbar: React.FC<TreeNodeFunctionsToolbarProps> =
   const [clickedInfo, setClickedInfo] = React.useState<FunctionIconInfo | undefined>(undefined);
 
   React.useEffect(() => {
-    getToolbarIcons(props.treeNodeIconMapper, props.selectedNodes, setClickedInfo)
-      .then((icons) => setToolbarIcons(icons))
+    getToolbarIcons(props.treeNodeIconMapper, props.selectedNodes, setClickedInfo) // eslint-disable-line @typescript-eslint/no-floating-promises
+      .then((icons) => setToolbarIcons(icons));
   }, [props.treeNodeIconMapper, props.selectedNodes]);
 
-  React.useMemo(() => {
+  React.useMemo(async () => { // eslint-disable-line @typescript-eslint/no-floating-promises
```

Comment:
for useMemo's that are not expected to return a value (just run asynchronously) a `void` operator can be added in front to indicate that it does not await the Promise on purpose, e.g.:
```
void React.useMemo(async () => {...}, [...]);
```
That way the linter will not complain